src/components/content-tab-overview.tsx

```diff
@@ -234,15 +237,43 @@ export function ContentTabOverview( { selectedSite }: ContentTabOverviewProps )
 					className={ cx(
 						'w-full min-h-40 max-h-60 overflow-hidden rounded-sm border border-a8c-gray-5 bg-a8c-gray-0 mb-2 flex items-center justify-center',
 						loading && skeletonBg,
-						isThumbnailError && 'border-none'
+						isThumbnailError && 'border-none',
+						siteRunning && 'hover:border-a8c-blueberry duration-300'
 					) }
 				>
 					{ isThumbnailError && ! loading && (
 						<div className="flex items-center justify-center w-full h-full leading-5 text-a8c-gray-50">
 							{ __( 'Preview unavailable' ) }
 						</div>
 					) }
-					{ ! loading && (
+					{ ! loading && siteRunning && (
+						<button
+							aria-label={ __( 'Open WP Admin' ) }
```

Comment:
I am finding a bit strange that the aria-label is `Open WP admin` but the text on the actual thumbnail is `Open site`. Should we consolidate the two to be consistent there? Perhaps the text on the thumbnail should be `Open WP admin` if that is what the action of clicking on the thumbnail does?

Reply:
I fixed it already when I changed the link to open site frontend instead of WP Admin.